db/schema.rb

```diff
@@ -282,6 +282,7 @@
     t.boolean "approved"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.datetime "disabled_at"
```

Comment:
Maybe we could add an index to `disabled_at` as well as it'll be in all queries in the end?

Reply:
Its a good shout, but I'm not sure its worth it given the sizes of these tables; the overhead of the index probably outweighs the benefit 🤷‍♂️

Reply:
Yeah agree.. just seen.. the tables size are near to "zero".. index won't make any difference. Tks